src/index.tsx

```diff
@@ -329,10 +331,21 @@ class Cropper extends React.Component<Props, State> {
     })
   }
 
+  calculateNewZoom(deltaY: number): number {
+    if (!this.props.logarithmicScrolling) {
+      return this.props.zoom - (deltaY * this.props.zoomSpeed) / 200
+    }
+    const zoomY = Math.log10(this.props.zoom)
+    const minZoomY = Math.log10(this.props.minZoom)
+    const maxZoomY = Math.log10(this.props.maxZoom)
+    const newZoomY = zoomY - (deltaY * this.props.zoomSpeed * (maxZoomY - minZoomY)) / 200
+    return Math.pow(10, newZoomY)
+  }
+
   onWheel = (e: WheelEvent) => {
     e.preventDefault()
     const point = Cropper.getMousePoint(e)
-    const newZoom = this.props.zoom - (e.deltaY * this.props.zoomSpeed) / 200
+    const newZoom = this.calculateNewZoom(e.deltaY)
```

Comment:
might want to instead divide here by 200 so that the deltaY for any other way of calling calcluateNewZoom isn't scaled by this pixel value, and is instead scaled in the natural zooming unit.